### PR TITLE
add process overview

### PR DESCRIPTION
**Pull request by johevemi**
## Description

Add process overview page

## References

- #0000

### CODE REVIEW

File: libs/dh/wholesale/feature-overview/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (BjarkeMeier) [inline libs/dh/wholesale/feature-overview/.eslintrc.json:1]: Doesn't look like something we like to need (and maintain) in every single feature folder.

File: libs/dh/wholesale/feature-overview/src/lib/dh-wholesale-overview.component.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (BjarkeMeier) [inline libs/dh/wholesale/feature-overview/src/lib/dh-wholesale-overview.component.scss:1]: Do we want empty files?

File: libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <ng-container *transloco="let transloco; read: 'wholesale'">
-  <h1>{{ transloco("startProcess") }}</h1>
-  <p>{{ transloco("processDescription") }}</p>
-  <watt-button>{{ transloco("start") }} </watt-button>
+  <h1>{{ transloco("startProcess.startProcessHeader") }}</h1>
+  <p>{{ transloco("startProcess.processDescription") }}</p>
+  <watt-button>{{ transloco("startProcess.startLabel") }} </watt-button>
```
Comment (BjarkeMeier) [inline libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.html:20]: nit:
```suggestion
  <watt-button>{{ transloco("startProcess.startButtonLabel") }} </watt-button>
```

**Review by BjarkeMeier — APPROVED** [review]

File: libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.html (hunk 3 above)
Comment (johevemi) [reply libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.html:20]: startLabel the rest of the translation

File: libs/dh/wholesale/feature-overview/src/lib/dh-wholesale-overview.component.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+/**
```
[the remark this answers is not in the document]
Comment (johevemi) [reply libs/dh/wholesale/feature-overview/src/lib/dh-wholesale-overview.component.ts:1]: that is the naming convention

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**